src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <span class="star__unactive"></span>
+      <span class="star__unactive"></span>
+      <span class="star__unactive"></span>
+      <span class="star__unactive"></span>
+      <span class="star__unactive"></span>
+    </div>
+    <div class="stars stars--1">
+      <span class="star__active"></span>
+      <span class="star__unactive"></span>
+      <span class="star__unactive"></span>
+      <span class="star__unactive"></span>
+      <span class="star__unactive"></span>
```

Comment:
![image](https://user-images.githubusercontent.com/80700247/211328838-a071c43b-1e92-4b51-bebc-35d5f858bd94.png)

add default background image to .stars__star class and add active image to modifiers + pseudo class :first-child and nth-child

If you have any questions - feel free to ask them in fe_chat